backend/src/api/models/user_groups/user_groups.py

```diff
@@ -66,21 +75,25 @@ def validate_user_public_key(cls, v, info: ValidationInfo) -> str:
             raise ValueError("Invalid group name format")
         return v
 
-    class Config:
-        extra = "forbid"
 
+class UserGroupUsersActions(UserGroupAction):
+    users: list[str]
 
-class AllUserGroupsOutput(BaseModel):
+    @field_validator("users")
+    def validate_user_public_key(cls, v, info: ValidationInfo) -> str:
```

Comment:
shouldn't this be called validate_emails?

Reply:
Done